examples/upload-files/README.MD

```diff
@@ -0,0 +1,25 @@
+# Select Files To Upload
+
+**Test Code**: [index.js](index.js)
+
+**Tested Page**: [index.html](index.html)
+
+TestCafe cannot test native browser dialogs, including those during file selection. For this reason, TestCafe utilizes another approach to file selection. This example shows how to select files to populate a `type="file"` input.
+
+The sample page includes a `<form>` that contains:
+
+* `<input>` element that holds the selected files
+* the `submit` button that sends the files to the server.
```

Comment:
`Submit button` is a button with type `submit` (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button).

`input with type=submit that sends the files to the server`
here and below.

Reply:
edited